src/elasticsearch/elasticsearch-sync.service.ts

```
@@ -0,0 +1,755 @@
+    filters?: Record<string, any>,
+    pagination?: { page: number; limit: number }
+  ): Promise<any> {
+    if (!isElasticsearchEnabled()) {
+      return { hits: { hits: [], total: { value: 0 } } };
+    }
+
+    try {
+      // Call searchUsers with proper parameters based on the service interface
+      return await this.userElasticsearchService.searchUsers(query);
+    } catch (error) {
+      LoggerUtil.error('Failed to search users in Elasticsearch', error, 'ElasticsearchSyncService');
```

ESLint

[error] Replace `'Failed·to·search·users·in·Elasticsearch',·error,·'ElasticsearchSyncService'` with `⏎········"Failed·to·search·users·in·Elasticsearch",⏎········error,⏎········"ElasticsearchSyncService"⏎······`

(prettier/prettier)